pkg/insight/insightstore/chunkstore.go

```diff
@@ -104,7 +104,7 @@ func PutChunksToCache(cache cache.Cache, chunks insight.Chunks) error {
 }
 
 func (s *store) getChunk(ctx context.Context, path string, kind model.InsightMetricsKind) (insight.Chunk, error) {
-	obj, err := s.filestore.GetObject(ctx, path)
+	obj, err := s.filestore.Get(ctx, path)
```

Comment:
nits, move this call after L120 (right before the returned value be used by obj.Content) could be better. It reduces one unnecessary call in case the kind check is failed.

Reply:
👍